examples/prompting/Basic_Reasoning.ipynb

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
<p>This link looks wrong - it has /quickstarts in it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>